docs/simpledroidble/usage.rst

```diff
@@ -9,39 +9,103 @@ This code is currently under active development and some features are not yet
 implemented or their API might change, but should be enough to help you get
 started.
 
+Obtain AAR dependencies
+=======================
+
+Using prebuilt AAR
+------------------
+
+1. Download the latest SimpleDroidBLE packages from the `releases page <https://github.com/OpenBluetoothToolbox/SimpleBLE/releases/>`_.
+
+2. Follow `Add SimpleDroidBLE to your project`_.
+
+Building from source
+--------------------
+
+1. Clone `the repo <https://github.com/OpenBluetoothToolbox/SimpleBLE>`_.
+
+2. Build the libraries using gradle (``./simpledroidble/gradlew assembleRelease``) or Android Studio. The following libraries will be built:
+    - ``simpledroidble/simpledroidble/build/outputs/aar/simpledroidble-release.aar`` 
+    - ``simpleble/src/backends/android/simpleble-bridge/build/outputs/aar/simpleble-bridge-release.aar``
```

Comment:
I would move the `simpleble-bridge` documentation to the `simpleble/usage.rst` page, as it would be specific for that kind of implementation.

Reply:
Isn't `simpleble-bridge.aar` required to use `simpledroidble`? If so, from the POV of the Android programmer, this should just be another requirement imo (i.e.: "You need both these AARs to use simpledroidble").